tests/test_person.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+from django.core.urlresolvers import reverse
+
+import pytest
+
+from lametro.views import LAMetroPerson
+
+
+@pytest.mark.django_db
+def test_person_page_redirects(client, metro_person, mocker):
+    person = metro_person.build()
+
+    # Patch a bunch of model attributes, because patching get_context_data
+    # breaks the url routing, for some reason.
+
+    mock_membership = mocker.patch('lametro.views.Membership', autospec=True)
+    mock_membership.role = 'Superhero'
+
+    mocker.patch.object(LAMetroPerson, 'latest_council_membership', return_value=mock_membership)
+
+    mocker.patch.object(LAMetroPerson, 'current_council_seat', return_value=None)
+    mocker.patch.object(LAMetroPerson, 'committee_sponsorships', return_value=[])
+
+    # Assert navigating to a valid slug works.
+
+    url = reverse('person', kwargs={'slug': person.slug})
+    rv = client.get(url)
+
+    assert rv.status_code == 200
+
+    # Assert navigating to a partially valid slug, redirects to a valid slug.
+
+    partial_slug = '-'.join(person.slug.split('-')[:-1])
```

Comment:
I think testing with a partial_slug works well....but why not test with a partially correct slug instead, since that would more closely mirror the "real-world" case?

```
# Something like....
old_uuid = str(uuid4())
old_slug = partial_slug + old_uuid.replace('-', '')[:13]
```